runtime/dali/src/weights/dex_router.rs

```diff
@@ -89,4 +89,14 @@ impl<T: frame_system::Config> dex_router::WeightInfo for WeightInfo<T> {
 			.saturating_add(T::DbWeight::get().reads(7 as Weight))
 			.saturating_add(T::DbWeight::get().writes(3 as Weight))
 	}
+	// Storage: DexRouter DexRoutes (r:1 w:0)
+	// Storage: Pablo Pools (r:1 w:0)
+	// Storage: Tokens Accounts (r:3 w:1)
+	// Storage: Tokens TotalIssuance (r:1 w:1)
+	// Storage: Pablo PriceCumulativeState (r:1 w:1)
+	fn remove_liquidity_single_asset() -> Weight {
+		(10_000 as Weight)
```

Comment:
Are you sure that it is the right weight? Or you just chose it for MVP?

Reply:
This file is generated automatically, I inserted these numbers manually for the reason to pass compiler checks (including CI checks). See comments about [`pablo`'s](https://github.com/ComposableFi/composable/blob/84ea1296041a8cb19d9d9caa26f8c8a00283ef86/frame/pablo/src/benchmarking.rs#L154) and [`dex-router`'s](https://github.com/ComposableFi/composable/blob/84ea1296041a8cb19d9d9caa26f8c8a00283ef86/frame/dex-router/src/benchmarking.rs#L261) benchmarks.

Reply:
Got it!